code/species_defs/species_defs.cpp

```diff
@@ -328,6 +328,34 @@ void parse_species_tbl(const char *filename)
 			if (optional_string("$Countermeasure type:"))
 				stuff_string(species->cmeasure_name, F_NAME, NAME_LENGTH);
 
+			if (optional_string("$Custom wingmate formations:")) {
+
+				while (check_for_string("(")) {
+					SCP_vector<vec3d> current_formation;
+
+					if (stuff_vec3d_list(current_formation) != MAX_SHIPS_PER_WING - 1) {
+						Warning(LOCATION, "A custom formation for species %s did not have %d positions. Ignoring.", species->species_name, MAX_SHIPS_PER_WING-1);
```

Comment:
```suggestion
					if (stuff_vec3d_list(current_formation) != MAX_SHIPS_PER_WING - 1) {
						Warning(LOCATION, "A custom formation for species %s did not have " SIZE_T_ARG " positions. Ignoring.", species->species_name, MAX_SHIPS_PER_WING-1);
```

Reply:
`MAX_SHIPS_PER_WING-1` here isn't a `size_t`, it is not specified (it is just `6-1`) so I think it gets type `int`. If you say it has type `size_t` then bad things happen unless `MAX_SHIPS_PER_WING-1` is also cast to type `size_t` (or `MAX_SHIPS_PER_WING` is defined to have a `size_t` cast in it). Is that what you mean?

Reply:
Hmm.  I was thinking "future proofing" in case we ever change `MAX_SHIPS_PER_WING` to `const size_t`, but maybe that's too clever by half.  You can either cast the argument to `size_t` or just leave the whole thing as-is.